src/features/messagebox/ui/MessageList.tsx

```
@@ -0,0 +1,114 @@
+      day: '2-digit',
+      hour: 'numeric',
+      minute: '2-digit',
+      hour12: true,
+    }
+
+    const formatter = new Intl.DateTimeFormat('ko-KR', options)
+    return formatter.format(e).replace(' ', '').replace(',', '')
+  }
+  const formattedDate = (createdAt: Date) => {
+    const date = new Date(createdAt)
+    if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)